scenarios/09_Hired_Swords.cfg

```diff
@@ -118,7 +123,7 @@
         side=1
         recruit=Dune Soldier,Dune Rover,Dune Burner
         gold=150
```

Comment:
dang it, how did I miss this one?! I definitely intended to make them all vary by difficulty...

Reply:
changed player gold to `{180 160 140}` in 4bdbac0; wasn't sure what to do with the enemy gold, though, so I'm not marking this one as resolved yet...

Reply:
> This can vary on difficulty, recommended {180 160 140}. As far as I can tell, you also didn't edit enemy gold, I think that can have slightly more variation on difficulty (e.g. increase by 15-20 per difficulty level instead of 10).

So, I'm giving this campaign another playthru, and... could you clarify a bit more which direction you were thinking of varying the enemy gold in? Which end is the one that should move? Possible options:
- `{100 115 130}` and `{90 105 120}`
- `{110 130 150}` and `{100 120 140}`
- `{100 120 140}` and `{90 110 130}`
- something else?